docs/guides/private-submodules.rst

```diff
@@ -0,0 +1,92 @@
+Using Private Git Submodules with Read the Docs for Business
+============================================================
+
+Read the Docs uses SSH keys (with read only permissions) in order to clone private repositories.
+An SSH key is automatically generated and added to your main repository, but not on your submodules.
+In order to give Read the Docs access to clone your submodules you'll need to add the public SSH key to each repository of your submodules.
+
+.. note::
+
+   You can manage which submodules Read the Docs should clone using a configuration file.
+   See :ref:`config-file/v2:submodules`.
+
+Project's SSH Key
+-----------------
```

Comment:
The steps to get the ssh key from the rtd project are repeated in all the next paragraphs, wasn't sure where else to put this.

Reply:
I think you can re-structure these section, by adding something like this after "In order to give Read the Docs access to clone your submodules you'll need to add the public SSH key to each repository of your submodules."

To do this:

1. Copy the public SSH key for your Read the Docs project
2. Add this SSH key to your submodule repositories

(here the note about you can manage which submodule...)

Now, the explanation of each of those two steps with titles:

"Copying the public SSH key"

(explanation about how to find it in your project and how to copy it)

"Adding the SSH key to your subdmodule"

"GitGub" (as subtitle)

"GitLab" (as subtitle)

etc...


----

This way, you have a 2 step path to follow, where each of them has its own section with detailed instruction, but you already have in mind that the total amount of work are 2 steps: Copying the key and Adding the key.

Reply:
The "Copying the key" is the same for all the providers. The "Adding the key" is the one that it's different on each provider.